lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1010,6 +1010,158 @@ module Resources
     }.freeze
 
     R4_RELATEDPERSON_CREATE ||= {
+      'resourceType': 'RelatedPerson',
```

Comment:
Let's remove all of the fields that we ignore to avoid confusion. Things like userSelected, displays/texts, array ids.

Reply:
+1

Reply:
array ids?

Reply:
updated: https://github.com/cerner/fhir.cerner.com/pull/689/commits/67460faa20177e8dab6a8154cd51a4930b8c74de#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1020-R1132